test/unit/lib/fastly-extended.test.js

```diff
@@ -0,0 +1,44 @@
+describe('fastly library', () => {
+    test('getLatestActiveVersion returns largest active VCL number', done => {
+
+        const mockedFastlyRequest = jest.fn((method, url, cb) => {
+            cb(null, [
+                {
+                    number: 4,
+                    active: false
+                },
+                {
+                    number: 1,
+                    active: true
+                },
+                {
+                    number: 2,
+                    active: true
+                },
```

Comment:
I don think this is good test data. Two versions cannot be active at the same time for a single service by definition no?

Reply:
Good point, changing!